buddyboss/buddyboss-platform
# Fixed - select2.min.css issue with rtl

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1959.

### How to test the changes in this Pull Request:

1. Change site language to any RTL language (Ex- he_IL - Hebrew)
2. Go to Edit Profile > Change Cover Photo
3. Check Console

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/creFoFVcPGM

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
